apps/portal/src/app/features/file-processor/file-processor.component.ts

```diff
@@ -13,9 +14,26 @@ export class FileProcessorComponent implements OnInit {
   // to do:the name will probably be changed to match the backend
   jobs: JobDetails[] = [];
 
-  constructor(private http: HttpClient) {}
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  summaries: any[] = [];
+
+  constructor(
+    private http: HttpClient,
+    private fileService: FileService,
+  ) {}
 
   ngOnInit(): void {
+    const jobIds = [1, 2]; // Once file upload api is integrated will change this code
+    this.fileService.fetchSummaries(jobIds).subscribe(
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      (result: any) => {
+        this.summaries = result.data.summaries.summaries;
+      },
+      // eslint-disable-next-line
+      (error) => {
+      },
```

Comment:
This should not be empty. Add toast message properly
Also never use just 
      // eslint-disable-next-line
It should always specify which error are we by passing.

Reply:
added a toast message for the error